db/migrate/20170207021033_add_phone_number_to_club_application.rb

```diff
@@ -0,0 +1,5 @@
+class AddPhoneNumberToClubApplication < ActiveRecord::Migration
+  def change
+    add_column :club_applications, :phone_number, :string
```

Comment:
This should be `text`. You should never use the `string` type in Postgres because it's limited to 255 chars.